CerebNet/data_loader/augmentation.py

```diff
@@ -179,9 +178,8 @@ def __call__(self, sample):
 
 class RandomFlip(object):
     """
-    Random horizontal flipping
+    Random horizontal flipping.
     """
-
     def __init__(self, cfg):
```

Comment:
Added cfg in docstring.
Could you provide the type and short desciption of cfg parameter?

I have written [MISSING] to avoid compilation warnings.

`    
def __init__(self, cfg):
        """
        Constructor function.

        Parameters
        ----------
        cfg : [MISSING]
            [MISSING].
        degrees : sequence or float or int,
             Range of degrees to select from.
             If degrees is a number instead of sequence like (min, max), the range of degrees
             will be (-degrees, +degrees).
        translate : tuple, float
             if translate=(a,b), the value for translation is uniformly sampled
             in the range
              -column_size * a < dx < column_size * a
              -row_size * b < dy < row_size * b
             If translate is a number then a=b=c.
             The value should be between 0 and 1.
        img_size : tuple
            img_size = (column_size, row_size)
        scale: tuple
            Range of min and max scaling factor.
        seed : int
            Random seed.

        Returns
        -------
        transform_mat : 3x3 matrix
            Random affine transformation.
        """
`